pkg/extender/extender.go

```diff
@@ -123,6 +125,24 @@ func (e *Extender) processFilterRequest(w http.ResponseWriter, req *http.Request
 	}
 
 	pod := args.Pod
+	for _, vol := range pod.Spec.Volumes {
+		// if any of pvc has restore annotation skip scheduling pod
+		if vol.PersistentVolumeClaim == nil {
+			continue
+		}
+		pvc, err := k8s.Instance().GetPersistentVolumeClaim(vol.PersistentVolumeClaim.ClaimName, pod.Namespace)
+		if err != nil {
+			msg := "Unable to find PVC %s" + vol.Name
+			storklog.PodLog(pod).Warnf(msg)
```

Comment:
I think the `Event` and `http.Error(...)` got removed when merging the other PR

Reply:
yes, thanks for pointing out. Do we need to check `nil` for pod.Spec here

Reply:
No, `PodLog()` checks for `nil`